packages/plugins/documentation/admin/src/pages/Settings.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+
+import { Main } from '@strapi/design-system';
+import { useAPIErrorHandler, Page, useNotification } from '@strapi/strapi/admin';
+import { useIntl } from 'react-intl';
+
+import { SettingsForm } from '../components/SettingsForm';
+import { useGetInfosQuery, useUpdateSettingsMutation } from '../services/api';
+import { getTrad } from '../utils';
+
+const SettingsPage = () => {
+  const { toggleNotification } = useNotification();
+  const { formatMessage } = useIntl();
+  const { formatAPIError } = useAPIErrorHandler();
+  const { data, isError, isLoading } = useGetInfosQuery();
+  const [updateSettings] = useUpdateSettingsMutation();
+
+  const onUpdateSettings = async (body: any) => {
```

Comment:
Should this handle server side validation errors or are we not that fussed for this?